src/app/_components/writingPad/writingPad.tsx

```diff
@@ -1,47 +1,180 @@
 "use client";
-
-import ReactQuill from "react-quill";
+import EditNoteIcon from "@mui/icons-material/EditNote";
 import "react-quill/dist/quill.snow.css";
 import { toolbarConfig } from "./config/configs";
 import { useContentForm } from "./hooks/useContentForm";
 import { Controller } from "react-hook-form";
+import { Box, Button, Typography } from "@mui/material";
+import FeedOutlinedIcon from "@mui/icons-material/FeedOutlined";
+import FolderIcon from "@mui/icons-material/Folder";
+import ChecklistIcon from "@mui/icons-material/Checklist";
+import { type EditorPost } from "~/app/editor/types/types";
+import dynamic from "next/dynamic";
+const ReactQuill = dynamic(() => import("react-quill"), { ssr: false });
 
-export function WritingPad() {
+type WritingPadProps = {
+  handleOpen: () => void;
+  editorPostData: EditorPost;
+};
+const WritingPad: React.FC<WritingPadProps> = ({
+  handleOpen,
+  editorPostData,
+}) => {
   const { handleSubmit, reset, control } = useContentForm();
 
   const onSubmit = (data: { content: string }) => {
     console.log(data);
   };
 
   return (
-    <div className="w-full">
-      {/* <h1>{JSON.stringify(users.data)}</h1> */}
-      <form onSubmit={handleSubmit(onSubmit)} className="flex flex-col gap-2">
-        <Controller
-          control={control}
-          name="content"
-          render={({ field }) => (
-            <ReactQuill
-              {...field}
-              modules={{
-                toolbar: toolbarConfig,
+    <Box
+      sx={{
+        width: "98%",
+        height: "100%",
+        display: "flex",
+        flexDirection: "column",
+      }}
+    >
+      <form
```

Comment:
We dont really need separate form tag here, we can simply call the handle submit of react-hook-form on click of submit button

Reply:
done